app/models/participant_declaration.rb

```diff
@@ -25,6 +25,8 @@ class ParticipantDeclaration < ApplicationRecord
            through: :billable_statement_line_items,
            source: :statement
 
+  has_many :outcomes, class_name: "ParticipantDeclarationOutcome"
```

Comment:
Outcomes are always going to be for NPQ declarations, is there a way we can namespace this to ensure that's evident?

Reply:
- moved it to `ParticipantDeclaration::Outcome::NPQ` (https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/95eaef1ea5c82c20e3500b50f90e3b5c105fdbb5 and https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/04c5b2128d33f93e8c250cf29db7bd320ff6647a)
- moved the `has_many` to `ParticipantDeclaration::NPQ` (https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/830d23e254ade372f9137296e22fc01d00e099ef)
- specify `ParticipantDeclaration::NPQ` for the `belongs_to` (https://github.com/DFE-Digital/early-careers-framework/pull/2755/commits/8a9330b7454e8c1680a3ef6b6e56d99caafe525f)